test/v1/connectors/DeleteBenefitAmountsConnectorSpec.scala

```diff
@@ -43,6 +43,18 @@ class DeleteBenefitAmountsConnectorSpec extends ConnectorSpec {
         result shouldBe outcome
       }
 
+      "the downstream call is successful, not tax year specific and the desIf_Migration feature-switch is turned on" in new IfsTest with Test {
+        def taxYear: TaxYear = TaxYear.fromMtd("2017-18")
```

Comment:
Can we add:
```
        MockedAppConfig.featureSwitches returns  Configuration("desIf_Migration.enabled" -> true)
```
in here to make it more explicit what the conditions for the test are.